gateway/modules/eventing/helpers.go

```diff
@@ -179,6 +179,22 @@ func (m *Module) generateProcessedEventRequest(eventID string) *model.UpdateRequ
 	}
 }
 
+func (m *Module) generateDLQEventRequest(eventDoc *model.EventDocument) error {
+	req := model.QueueEventRequest{
+		Type: "dlq_event_trigger_" + eventDoc.RuleName,
```

Comment:
Use fmt.sprintf() & store the "dlq_event_trigger_" as a contant in utils/ constants.go file. name it as DLQEventTriggerSuffix